src/main/java/org/broadinstitute/consent/http/service/UserService.java

```diff
@@ -211,7 +211,7 @@ public List<User> getUsersAsRole(User user, String roleName) {
                     throw new NotFoundException("Signing Official (user: " + user.getDisplayName() + ") is not associated with an Institution.");
                 }
             case Resource.ADMIN :
-                return new ArrayList<>(userDAO.findUsers());
+                return new ArrayList<>(userDAO.findUsersWithLCsAndInstitution());
```

Comment:
If `userDAO.findUsersWithLCsAndInstitution()` returned a list, we wouldn't need this construction.